### ISSUE TITLE
Resolve incorrect tabindex values in IE

### ISSUE THREAD

**Issue by basham** [iuux/adrx-quicknotes#48]
IE returns 0 instead of -1 for elements without an explicit `tabindex` yet should be non-focusable (like `<div>`). This prevents the keyboard trapping in overlays and dropdowns from working correctly.

Details regarding the issue:
http://nemisj.com/focusable/
